src/routes/vendors.js

```diff
@@ -56,5 +56,20 @@ router.get(
     }
   }
 );
+/**
+ * GET Vendor Stats
+ */
+router.get('/count', authenticate(), async (req, res) => {
+  log.info(`request for vendor stats`);
+  try {
+    const result = await Vendor.count();
+
+    res.type('application/json');
+    res.json(result);
```

Comment:
I am not sure where I stand on this ultimately, but maybe we should start a pattern where all endpoints that return JSON return at least an object with the payload inside - my rationale is ultimately related to error handling. If there are ever any scenarios where an API call returns _either_ a number _or_ an object then we might end up with some ugly code on the frontend that has to figure out what type of response we got based on its type.

Thoughts?

Reply:
I've updated this PR and the /diluents PR to match the stats on the dashboard home where the count returned object is `{ [type] : [count] }`.

I am fairly certain maintaining consistent patterns will be appreciated down the road. I'll begin migrating the others as I update the frontend code.